build_scripts/windows/scripts/build.cmd

```diff
@@ -187,7 +187,7 @@ msbuild /t:rebuild /p:Configuration=Release %REPO_ROOT%\build_scripts\windows\az
 
 if %errorlevel% neq 0 goto ERROR
 
-start %OUTPUT_DIR%
+echo %OUTPUT_DIR%
```

Comment:
What's purpose to change "start" to "echo"?

Reply:
https://github.com/Azure/azure-cli/pull/24201#discussion_r996535656

Reply:
@jiasli,
I felt that it was not necessary to open this directory in the last step, so I changed it to the echo command.

Reply:
You are right. I don't feel the necessity either.